<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>970</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>561</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>791</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Série</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;Titre de la série&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Notes:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre de saisons:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>60</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Liste des séries: </string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>791</width>
     <height>251</height>
    </rect>
   </property>
   <property name="title">
    <string>Saisons</string>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>301</width>
      <height>231</height>
     </rect>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderVisible">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Titre</string>
     </property>
    </column>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>30</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Titre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;Titre de la saison&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>60</y>
      <width>57</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Date</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>60</y>
      <width>181</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;Date de la saison&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="open_folder_button">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Ouvrir le dossier de la saison</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/icons/folder-horizontal-open.png</normaloff>../resources/icons/folder-horizontal-open.png</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="show_view_history_button">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>210</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Historique de visionnage</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../resources/icons/film-timeline.png</normaloff>../resources/icons/film-timeline.png</iconset>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="search_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Filtrer les éléments de la liste</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>130</y>
     <width>31</width>
     <height>91</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="add_serie_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/plus.png</normaloff>../resources/icons/plus.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="edit_serie_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/pencil.png</normaloff>../resources/icons/pencil.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_serie_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/minus.png</normaloff>../resources/icons/minus.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>380</y>
     <width>31</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QPushButton" name="add_season_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/plus.png</normaloff>../resources/icons/plus.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="edit_season_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/pencil.png</normaloff>../resources/icons/pencil.png</iconset>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_season_button">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>../resources/icons/minus.png</normaloff>../resources/icons/minus.png</iconset>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="view_deleted_elements_button">
   <property name="geometry">
    <rect>
     <x>700</x>
     <y>50</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher les séries / saisons supprimées</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>effacer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
